test/hotspot/jtreg/vmTestbase/nsk/share/jdi/ClassExclusionFilterTest.java

```diff
@@ -38,7 +37,10 @@ public class ClassExclusionFilterTest extends EventFilterTest {
     protected String[] classPatterns;
 
     public static void main(String[] argv) {
-        System.exit(run(argv, System.out) + Consts.JCK_STATUS_BASE);
+        int result = run(argv,System.out);
+        if (result != 0) {
+            throw new RuntimeException("TEST FAILED with result " + result);
+        }
```

Comment:
Why do we have tests under jdi/share?

Reply:
It is not a test, but helper class with not the best name.